### DISCUSSION TITLE
Requesting test funds: 5Fes3ciLtT8nRF3RwYsMYRCBzmNAxbkeExsmkRbVWoWVbET

### DISCUSSION THREAD

**Discussion by mikhailivich** [entropyxyz/community#56] [Get Test Funds] [answered]
### Address

5Fes3ciLtT8nRF3RwYsMYRCBzmNAxbkeExsmkRbVWoWVbET

**Comment by johnnymatthews** [comment, marked as answer]
It looks like your missing a character. Substrate/Entropy addresses are 48 characters long; the address you've supplied is 47 characters long.

**Reply by mikhailivich** [reply]
> It looks like your missing a character. Substrate/Entropy addresses are 48 characters long; the address you've supplied is 47 characters long.

Yes, i'm sorry, this is my correct addres - 5Fes3ciLtT8nRF3RwYsMYRCBzmNAxbkeEoxsmkRbVWoWVbET

**Reply by johnnymatthews** [reply]
Sent!
